app/src/main/java/com/iic/shopingo/ui/trip_flow/activities/ManageTripActivity.java

```diff
@@ -92,11 +98,11 @@ public int getCount() {
   }
 
   @Override
-  public void onRequestAccepted(Request request) {
+  public void onRequestAccepted(IncomingRequest request) {
     ShoppingList sl = new ShoppingList();
-    sl.requesterName = request.name;
-    sl.phoneNumber = request.location.phone;
-    for (String itemTitle : request.items) {
+    sl.requesterName = request.getRequester().getFirstName();
+    sl.phoneNumber = request.getRequester().getPhoneNumber();
```

Comment:
What impact does repeating all these `getRequester().getX()` and `getShoppingList().getY()` have?